src/praisonai/praisonai/cli/commands/permissions.py

```
@@ -0,0 +1,269 @@
+    """Manage tool execution permissions and approval rules."""
+    pass
+
+
+def get_project_permissions_dir() -> str:
+    """Get the project-scoped permissions directory."""
+    project_dir = os.getcwd()
+    return os.path.join(project_dir, ".praisonai", "permissions")
+
+
+@permissions.command()
+def list():
```

Ruff (0.15.17)

[error] Variable `list` is shadowing a Python builtin

(A001)